README.md

```diff
@@ -24,6 +24,8 @@ Where opts is an object with the following properties (all optional):
 * **port** - the desired port for the server to use. Defaults to `8000`.
 * **dir** - the directory to launch from (where it should look for a Cordova project). Defaults to cwd.
 * **simhostui** - the directory containing the UI specific files of the simulation host. Defaults to the bundled simulation host files, found in `src/sim-host/ui`.
+* **nolivereload** - A boolean. Set to true to disable live reload. Defaults to false.
+* **forceprepare** - A boolean. Set to true to force a `cordova prepare` whenever a file changes during live reload. If this is false, the server attempts to simply copy the changed file to the platform rather than doing a `cordova prepare`. Ignored if live reload is disabled. Defaults to false.
```

Comment:
If this configuration is only for live reload functionality, maybe we can define it in a single configuration object in where enabling live reload and forcing prepare can be defined:
```
livereload: {
    enabled: true,
    forceprepare: false
};
```